diff --git a/lib/main.js b/lib/main.js
index 87aa87b00cf9eb27dcad88b11b2570298cc8c422..a834bc272cc327d89055607809aafa4473113055 100644
--- a/lib/main.js
+++ b/lib/main.js
@@ -771,7 +771,7 @@ function createChannel(streamIn) {
         });
       }
     };
-    if ((typeof input === "string" || input instanceof Uint8Array) && input.length > 1024 * 1024) {
+    if ((typeof input === "string" || input instanceof Uint8Array) && input.length > 1073741824) {
       let next = start;
       start = () => fs3.writeFile(input, next);
     }
